d2/src/main/java/com/linkedin/d2/balancer/simple/SimpleLoadBalancer.java

```diff
@@ -943,8 +941,7 @@ private void handleTimeoutFromGetClusterAndUriProperties(Throwable e, String clu
     }
     else
     {
-      clusterAndUriPropertiesCallback.onError(
-          new ServiceUnavailableException(clusterName, "PEGA_1011. " + e.getMessage(), e));
+      _log.error("getClusterAndUriProperties for {} timed out, but no value in cache!", clusterName);
```

Comment:
There's already a warn log in `getClusterAndUriPropertiesFromCache`:
```java
      warn(_log, "unable to find cluster: ", clusterName);
```

Can we remove that log and change the level of this one to warn?

Reply:
Discussed with @shivamgupta1 . As this log is in `getClusterAndUriProperties for timeout` scenario, so it's best to keep this to check distinguish this special scenario. But it's ok to change the level from `error` to `warn`.